bin/cron/ops_data_pull

```diff
@@ -21,7 +39,8 @@ pd_scripts.each do |script|
                    :'Ops First Name' => user.ops_first_name,
                    :'Ops Last Name' => user.ops_last_name,
                    :'Email' => user.email,
-                   :'District' => user.district.try(:name) || 'None'
+                   :'District' => user.district.try(:name) || 'None',
+                   :'School' => user.school,
```

Comment:
Changing to 'None' for consistency. Also changed to user.ops_school rather than user.school as the former seems to be better populated.